libcxx/utils/libcxx/test/features.py

```diff
@@ -13,9 +13,12 @@
 import subprocess
 import sys
 
-_isClang = lambda cfg: "__clang__" in compilerMacros(cfg) and "__apple_build_version__" not in compilerMacros(cfg)
+_isAnyClang = lambda cfg: "__clang__" in compilerMacros(cfg)
```

Comment:
I would much prefer if we had a way of detecting `cl-style-warnings` here. I am waging a war against stale features in our test suite, and `cl-style-warnings` without any way to detect it looks exactly like a stale Lit feature.